features/step_definitions/metadata.js

```diff
@@ -33,31 +33,24 @@ module.exports = function server() {
             throw new Error('jobName is neither BAR or BAZ');
         }
 
-        request({
+        return request({
             uri: `${this.instance}/${this.namespace}/jobs/${jobId}/builds`,
             method: 'GET',
             json: true
         }).then((response) => {
             this.buildId = response.body[0].id;
-            this.meta = response.body.meta;
 
             Assert.equal(response.statusCode, 200);
         });
     });
 
-    this.Then(/^add the { "foo": (.*) } to metadata in the "main" build container$/, () => null);
+    this.Then(/^add the { "foo": (?:.*) } to metadata in the "main" build container$/, () => null);
 
-    this.Then(/^add the { "bar": (.*) } to metadata in the "BAR" build container$/, () => null);
+    this.Then(/^add the { "bar": (?:.*) } to metadata in the "BAR" build container$/, () => null);
 
-    this.Then(/^in the build, the { "foo": "(.*)" } is available from metadata$/, (value) => {
-        Assert.ok('foo', this.meta);
```

Comment:
It seems that `meta` can not be obtained from the API while the status is RUNNING. So, we can't testing in this method that is running a build.